solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -91,16 +91,18 @@ library DKG {
     /// @dev Size of a group in the threshold relay.
     uint256 public constant groupSize = 64;
 
-    /// @dev Minimum number of group members needed to interact according to the
-    ///      protocol to produce a relay entry.
+    /// @dev The minimum number of group members needed to interact according to
+    ///      the protocol to produce a relay entry. The adversary can not learn
+    ///      anything about the key as long as it does not break into
+    ///      groupThreshold+1 of members.
     uint256 public constant groupThreshold = 33;
```

Comment:
Shouldn't we leave a constant named `signatureThreshold` but just change its value to 33? We could remove `groupThreshold` constant then.